docs/account_management/cheaha_account.md

```diff
@@ -57,11 +57,14 @@ Mistakes happen, and sometimes what we thought we programmed wasn't quite what w
 
 Other reasons for holds include, but are not limited to:
 
-- If your account shows no logged usage of our resources for an extended period
+- If your account shows no logged usage of our resources for an extended period (at least one year)
 - Misuse (intentional or not) of Research Computing resources
```

Comment:
As long as you are on this page, can you please remove the parenthetical part on this line? I don't like the language I wrote previously.